index.js

```diff
@@ -29,6 +30,23 @@ class ResembleHelper extends Helper {
     return folderPath;
   }
 
+  resolveRelativePath(folderPath) {
```

Comment:
**suggestion:** I don't think this function is meant to be public, that's why given how this file is written, it might be great to prepend a `_`.